presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestExternalHiveTable.java

```diff
@@ -43,6 +45,17 @@ public Requirement getRequirements(Configuration configuration)
                 mutableTable(NATION_PARTITIONED_BY_BIGINT_REGIONKEY));
     }
 
+    @Test
+    public void testAnalyzeExternalTable()
+    {
+        TableInstance nation = mutableTablesState().get(NATION.getName());
+        onHive().executeQuery("DROP TABLE IF EXISTS " + EXTERNAL_TABLE_NAME);
+        onHive().executeQuery("CREATE EXTERNAL TABLE " + EXTERNAL_TABLE_NAME + " LIKE " + nation.getNameInDatabase());
+
+        // We allow to run ANALYZE on external tables as long as the user has the privileges.
```

Comment:
does Hive allow that too?

Reply:
Yes. Will add a test in `TestExternalHiveTable.java` to test `SHOW STAT`.